index.org

```diff
@@ -90,6 +114,40 @@ sur la licence d’un logiciel avec la technique précendente, il faut se
 rendre sur le dépôt de code source du logiciel. La licence doit se
 trouver soit à la racine du dépôt soit dans un répertoire =LICENSES=.
 
+*** Analyser : les autres aspects 
+
+Si la licence est l'élément indispensable que l'on vérifie 
+avant d'aller plus loin, il en est trois autres à ne pas négliger : les fonctionnalités, la sécurité et la communauté. 
+
+**** Fonctionnalités 
+
+Il n'est pas rare qu'un logiciel ne couvre pas entièrement les besoins fonctionnels visés. 
+Et c'est là que la magie opère : vous pouvez [[#contribuer-a-un-logiciel-libre][contribuer]] 
+au projet et ainsi participer à l'effort collectif, récompenser la communauté. 
+
+Il est donc utile de distinguer au sein des fonctionnalités attendues, 
+celles qui sont couvertes et celles qui ne le sont pas. Parmi ces 
+dernières, il est utile de mettre en balance leur criticité pour vous 
+et leur moyen d'intégration dans le projet (ex. "Est-ce facile à intégrer 
+via des plugins ou cela nécessite-t-il une modification du coeur ?"). 
+
+**** Sécurité 
```

Comment:
Peut-être donner quelques questions à se poser pour vérifier qu'un logiciel libre est « sécurisé » ? en précisant qu'il y a des techniques industrielles solides pour aborder ce sujet ?

Reply:
I've tried something but we could switch to temp.org if it's not enough ;)

Reply:
Good enough for now :)